Java/Dijkastra Shortest Path Algorithm (Graph in form of Adjacency Matrix)/DijkastraShortestPathAlgorithm.java

```diff
@@ -0,0 +1,241 @@
+/* Copyright 2018 Aman Mehara
```

Comment:
Please write your own name and change the year to 2020.

Reply:
Thanks, This correction has now been taken care.